src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -45,6 +48,13 @@ def channel_dict(self):
                 f'{self.channel_prefix}/list_bids': self._list_bids,
                 }
 
+    @property
+    def filtered_bids(self):
```

Comment:
Just to be more verbose:
```suggestion
    def filtered_bids_next_market(self):
```

Reply:
Will do! Out of curiosity: why is it called `next_market` rather than `current_market`? Even the docstring in the method refer to it as "current market":
https://github.com/gridsingularity/d3a/blob/cc260bac3905416325ee72b5253f21f15ef0e308/src/d3a/models/area/__init__.py#L502-L507

Reply:
Yes, this is a common confusion for "beginners" that we couldn't solve until today. Happy about any suggestions!

There are two "nows" in energy markets and this what is confusing from my experience.
 Ok, let's say it is 11:08, so the market_slot, where the assets are trading in is 11:15-11:30. This is in the future, so it could be called "next_market" although it is really the "current_market" because the marke where all trades where done for the current time (11:00-11:15) is already closed. 
So there is this chain of markets
[..., past_market_1, _past_market_2_, **current_market**, future_market_1, future_market_2, ...]
with the times:
[..., 10:45-11:00, _11:00-11:15_, **11:15-11:30**, 11:30-11:45, 11:45-12:00, ...]
The market at the current time is the _past_market_2_ and the market that is currently held is  **current_market**
So the latter was formally named next_market because it is in the future, but really it is the currently held market.

Sorry to be so verbose. May even confuse you more.
Maybe the others have a better way of explaining

Reply:
Ok, so if I understand correctly there is something like:
- "ongoing" market (`past_market_2` in your example): that is the market where the trades I've already posted in the past are being applied right now . 
- "next" market (`current_market` in your example): the market in which I can place my bids/offers now (but it will start when the ongoing market's execution will end).

It's called "current" because the backend is taking bids/offers for it "now", even though it will start at the end of the current slot. I hope this was correct :)

Reply:
> Will do! Out of curiosity: why is it called `next_market` rather than `current_market`? Even the docstrings in the method refer to it as "current market":
> https://github.com/gridsingularity/d3a/blob/cc260bac3905416325ee72b5253f21f15ef0e308/src/d3a/models/area/__init__.py#L502-L507

the docstring seems to be decoupled with method name, what actually has to be settled is if for the most immediate future open market which is accepting offers/bids and simultaneously clearing it `trades` should be called `current_market` or the `next_market`. If the term `current_market` to be used for the most immediate future open market then what should the market where all settled trades are currently getting realised be called